content/en/continuous_integration/setup_tests/_index.md

```diff
@@ -3,13 +3,7 @@ title: Set up Tracing on CI Tests
 kind: documentation
 ---
 
-To gather test suite results, performance, and reliability data, first set up the Agent, and then instrument your tests:
-
-{{< whatsnext desc="1. Setting up and configuring the Agent:" >}}
-    {{< nextlink href="continuous_integration/setup_tests/agent" >}}Installing the Datadog Agent{{< /nextlink >}}
-    {{< nextlink href="continuous_integration/setup_tests/containers" >}}Forwarding Environment Variables for Tests in Containers{{< /nextlink >}}
```

Comment:
It looks like the link for the Containers was removed here, but the page still exists, was this on purpose?